<ui version="4.0" >
 <class>Config</class>
 <widget class="QWidget" name="Config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bespin Configuration</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Buttons</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Gradient</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="gradButton" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Checkbox Appereance</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="checkMark" >
         <item>
          <property name="text" >
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>V</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>O</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QCheckBox" name="fullButtonHover" >
         <property name="text" >
          <string>Hover fill to border</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="sunkenButtons" >
         <property name="text" >
          <string>Sunken frame (exp)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QCheckBox" name="showScrollButtons" >
         <property name="text" >
          <string>Show Scrollbar Buttons</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>191</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Tabbars</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Gradient</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="gradTab" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Background</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="crTabBar" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Active Item</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="crTabBarActive" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Transition</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QComboBox" name="tabTransition" >
         <item>
          <property name="text" >
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Scanline blend</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Slide in</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Slide out</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Roll in</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Roll out</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Open vertically</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Close vertically</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Open horizontally</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Close horizontally</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Crossfade</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>211</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite" >
      <attribute name="title" >
       <string>Menus</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Selected Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="crMenuActive" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Gradient</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="gradMenuItem" />
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Popups</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="crPopup" />
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="showMenuIcons" >
            <property name="text" >
             <string>Show icons</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QCheckBox" name="menuShadow" >
            <property name="text" >
             <string>"Shadow" outline</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>151</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_2" >
      <attribute name="title" >
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Progress color role</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="crProgressBar" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>Progress Gradient</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="gradProgress" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_12" >
         <property name="text" >
          <string>Combo Gradient</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="gradChoose" />
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>151</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_3" >
      <attribute name="title" >
       <string>Manage</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="7" >
        <widget class="QListWidget" name="store" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="resizeMode" >
          <enum>QListView::Adjust</enum>
         </property>
         <property name="uniformItemSizes" >
          <bool>true</bool>
         </property>
         <property name="selectionRectVisible" >
          <bool>true</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="btnRestore" >
         <property name="text" >
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="btnStore" >
         <property name="text" >
          <string>Store...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3" >
        <widget class="QPushButton" name="btnDelete" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>85</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>85</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>85</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5" >
        <widget class="QPushButton" name="btnImport" >
         <property name="text" >
          <string>Import...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6" >
        <widget class="QPushButton" name="btnExport" >
         <property name="text" >
          <string>Export...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item rowspan="2" row="0" column="2" >
    <widget class="QTextBrowser" name="info" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Background Mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="bgMode" >
     <item>
      <property name="text" >
       <string>Plain</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Scanlines</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Complex</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Vertical Top/Bottom Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Horizontal Left/Right Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Vertical Center Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Horizontal Center Gradient</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
